tensorflow/tools/ci_build/linux/rocm/run_gpu_single.sh

```diff
@@ -27,7 +27,7 @@ echo "Bazel will use ${N_BUILD_JOBS} concurrent build job(s) and ${N_TEST_JOBS}
 echo ""
 
 # First positional argument (if any) specifies the ROCM_INSTALL_DIR
-ROCM_INSTALL_DIR=/opt/rocm-5.1.0
+ROCM_INSTALL_DIR=/opt/rocm-5.2.0
```

Comment:
Should this be also changed to rocm-5.3.0?

Reply:
@akuegel Yes, just pushed an update.

Thanks!